packages/nx-bun/src/executors/build/schema.d.ts

```diff
@@ -1,6 +1,8 @@
 import type { BaseBun } from '../../utils/common-options'
 
 export interface BundleExecutorSchema extends BaseBun {
+  main: string;
```

Comment:
main? I think here you are referring to the first file? If so you need entrypoints. It's the naming convention from bun as build can support multiple entries

Reply:
@Jordan-Hall hmm, I see. It should be optional then.
If main is not present in the build target config, it should then be present in the run target config?